core/Router/WP_Router.php

```
@@ -29,6 +29,31 @@ public static function register( $routes = [] ) {
+
+	/**
+	 * Disable error display for this plugin's REST requests so stray PHP
+	 * notices/deprecations never leak into the JSON response body.
+	 *
+	 * @param  mixed            $result
+	 * @param  \WP_REST_Server  $server
+	 * @param  \WP_REST_Request $request
+	 *
+	 * @return mixed
+	 */
+	public static function silence_error_display( $result, $server, $request ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$server'. (undefined)

(UnusedFormalParameter)